registry/txt.go

```diff
@@ -397,3 +431,12 @@ func (im *TXTRegistry) removeFromCache(ep *endpoint.Endpoint) {
 		}
 	}
 }
+
+func isManagedRecord(record *endpoint.Endpoint, managedTypes []string) bool {
+	for _, mt := range managedTypes {
+		if record.RecordType == mt {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
There is a similar to a method defined in `plan.go`: 

```
func isManagedRecord(record string, managedRecords []string) bool {
	for _, r := range managedRecords {
		if record == r {
			return true
		}
	}
	return false
}
```

Considering that `registry` is importing `plan`, we could be exporting the other method and reuse that.

Reply:
Reused.